packages/app-store/zohocalendar/api/callback.ts

```diff
@@ -46,14 +54,14 @@ async function getHandler(req: NextApiRequest, res: NextApiResponse) {
 
   const query = stringify(params);
 
-  const response = await fetch(`${OAUTH_BASE_URL}/token?${query}`, {
+  const response = await fetch(`${getOAuthBaseUrl(server_location || "com")}/token?${query}`, {
```

Comment:
We need to handle when "US" is returned as this will add "us" to the domain when it should be "com"